### ISSUE TITLE
set bootstrap class in a form field SplitPhoneNumberField() 

### ISSUE THREAD

**Issue by bikash829** [stefanfoulis/django-phonenumber-field#624]
Hi! I am trying to make a form using SplitPhoneNumberFiled(). Everything is working so well but i want to style those fields. And i couldn't find out how to set an attributes in a field. 

```python 
 phone = SplitPhoneNumberField()
    additional_phone = SplitPhoneNumberField()
    class Meta:
        model = User
        fields = ['phone','additional_phone']
```

That's my code.

**Comment by francoisfreitag** [comment]
You can customize the attrs through subclassing, by using https://github.com/stefanfoulis/django-phonenumber-field/blob/main/phonenumber_field/formfields.py#L134-L146.

This is hinted at in the `Usage` section of the documentation, but those hooks could be made public. I’m leaving the issue open to document those hooks.

**Comment by ilkkao** [comment]
yes, would be good to document. I just tried to make something like

```
    phone_number = SplitPhoneNumberField(
        required=True,
        widget=PhoneNumberPrefixWidget(
            country_attrs={'class': 'form-control select2'},
            number_attrs={'class': 'form-control',  'style': 'width: 600px;'},
        )
    )
```

to work but noticed eventually it's not possible anymore

**Comment by francoisfreitag** [comment]
Thanks for your input. I believe #631 should address your issues. Could you take a look at it and providing feedback?
The doc is available at https://django-phonenumber-field--631.org.readthedocs.build/en/631/reference.html#splitphonenumberfield.
